paperqa/agents/task.py

```diff
@@ -155,7 +158,10 @@ async def step(
             self.state.session.answer
         )
         if evaluation_callback := self._evaluation_callback:
+            if not isinstance(evaluation, MultipleChoiceEvaluation):
+                raise ValueError("evaluation has to be a MultipleChoiceEvaluation")
```

Comment:
Can you make this message be more precise (e.g. state why, so if anyone hits this they have some instructions) and a full sentence with capitalizations and a period? And in the `ValueError` use an f-string to state what type it actually was.

Also, don't restate `MultipleChoiceEvaluation` as a string literal, because we often rename classes, and IDE-based renames will miss any string literals. Use `f"{MultipleChoiceEvaluation.__name__}"`

Let's hold ourselves to a higher standard where possible